modules/apps/segments/segments-web/src/main/resources/META-INF/resources/js/components/criteria_builder/ContributorsBuilder.es.js

```diff
@@ -184,341 +150,133 @@ class ContributorBuilder extends React.Component {
 
 							<ClayLayout.ContainerFluid>
 								<div className="content-wrapper p-4">
-									{Liferay.FeatureFlags['LPS-166954'] ? (
-										<>
-											<ClayPanel
-												className="mb-4"
-												collapsable
-												defaultExpanded={true}
-												displayTitle={
-													<ClayPanel.Title>
-														<h2 className="m-0 text-dark">
-															{Liferay.Language.get(
-																'scope'
-															)}
-														</h2>
-													</ClayPanel.Title>
-												}
-												displayType="secondary"
-												showCollapseIcon
-											>
-												<ClayPanel.Body className="p-4">
-													{this.props
-														.siteItemSelectorURL && (
-														<div className="align-items-center d-flex justify-content-between mb-3">
-															<p className="mb-0 mr-6 text-dark">
-																{Liferay.Language.get(
-																	'select-the-scope-of-your-segment-to-specify-where-it-can-be-used'
```

Comment:
Could this translation be removed from the language.properties file?

Reply:
True, thank you!